src/main/java/com/eternalcode/parcellockers/deliverycode/repository/DeliveryCodeRepositoryImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.eternalcode.parcellockers.deliverycode.repository;
+
+import com.eternalcode.parcellockers.database.AbstractDatabaseService;
+import com.eternalcode.parcellockers.deliverycode.DeliveryCode;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public class DeliveryCodeRepositoryImpl extends AbstractDatabaseService implements DeliveryCodeRepository {
+
+    private final Map<UUID, DeliveryCode> cache = new HashMap<>();
```

Comment:
ja bym cache nie dodawał dopóki nie będzie realnego usecase gdzie będzie to potrzebne

Reply:
Będzie realny use case bardzo niedługo, na razie zostawiam, w razie co to usunę